app/assets/javascripts/custom.js

```diff
@@ -11,12 +11,12 @@ $(document).ready(function() {
                           }
                         };
       createGame(gameParams);
+      window.location = "/" + gameParams.game.name;
     } else {
       $('#welcome').append('Invalid wifi password. Try again.').css('background-color', '#ffcccc');
```

Comment:
Instead of injecting CSS directly here, could you create a class and just addClass with the CSS elements you want here?